### ISSUE TITLE
lib\Varien\File\Uploader.php:259

### ISSUE THREAD

**Issue by eWholesaler** [Inchoo/Inchoo_PHP7#111]
This fixed worked for me. Wasn't sure if you wanted to update this into your code.
https://magento.stackexchange.com/questions/105550/how-can-solve-error-magento-1-xx-on-php-7/105604#105604

File uploading will not work. Magento extends Mage_Core_Model_File_Uploader from above class, so we need to override this class and rewrite _validateFile() function replace below line

```
$params['object']->$params['method']($this->_file['tmp_name']);
with

$params['object']->{$params['method']}($this->_file['tmp_name']);
```

**Comment by nicanaca0** [comment]
+1, had this issue too and the fix above worked for me. I'm leaving my fix in place for now but would be nice if this module could include this fix.

(When I clicked to upload it just gets stuck on "Uploading...")

**Comment by Blackskyliner** [comment]
This seems to be already fixed in the latest Magento 1.9.3.x, see the 2.1.0 Branch of this repository where the overwrite of the models and the specified class was removed from the repository because of the fixes. I just looked in my installation 'lib/Varien/File/Uploader.php' and it was already fixed there too.
